### ISSUE TITLE
[BUG] Error [ERR_PACKAGE_PATH_NOT_EXPORTED] with 0.2.0

### ISSUE THREAD

**Issue by rogargon** [deltaDAO/nautilus#41]
## Summary

Upgraded Nautilus to 0.2.0 in a running project. Now not able to execute without any further change to the original code.

## Current Behavior

Getting error:

`Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/src/@types' is not defined by "exports" in .../nautilus/node_modules/@deltadao/nautilus/package.json`

By the way, `dist/src/index.d.ts` content is:

````javascript
export * from './access';
export * from './compute';
export * from './publish';
export * from './Nautilus';
````

## Environment

"@deltadao/nautilus": "^0.2.0",
"dotenv": "^16.3.1",
"ts-node": "^10.9.1",
"typescript": "^5.0.2",
"web3": "^1.9.0"

**Comment by moritzkirstein** [comment]
Hi @rogargon,

thanks for raising the issue. We released a fix with [0.2.1](https://github.com/deltaDAO/nautilus/tree/v0.2.1) that allows importing types as intended via 
```ts
import ... from '@deltadao/nautilus'
```
Let us know if this resolves your issue, otherwise it would be helpful to get some more information, e.g. on the import statement that causes the error, as we could not replicate the issue on our end.

**Comment by rogargon** [comment]
Yes, the import works now. Thanks!
